.github/workflows/test_build_deploy.yml

```diff
@@ -74,15 +74,17 @@ jobs:
         uses: c-hive/gha-yarn-cache@v1
 
       - name: Authenticate git clone
+        if: env.DISTRIBUTION == 'wire' || github.event_name != 'pull_request'
         env:
           GH_TOKEN: ${{secrets.OTTO_THE_BOT_GH_TOKEN}}
-        run: echo -e "machine github.com\n  login ${GH_TOKEN}" > ~/.netrc
+        run: echo "machine github.com login ${GH_TOKEN}" > ~/.netrc
 
       - name: Install JS dependencies
+        if: env.DISTRIBUTION == 'wire' || github.event_name != 'pull_request'
         run: yarn --frozen-lockfile
 
       - name: Test
-        if: env.DISTRIBUTION == 'wire'
+        if: env.DISTRIBUTION == 'wire' || github.event_name != 'pull_request'
```

Comment:
No tests for PRs?

Reply:
Why should we test everything twice? We are already testing with `DISTRIBUTION = wire` :)

Reply:
I don't get it what are we testing twice?

Reply:
Resolved in 1:1